arch/risc-v/include/irq.h

```diff
@@ -572,7 +572,7 @@ struct xcptcontext
    * another signal handler is executing will be ignored!
    */
 
-  uintptr_t *saved_regs;
+  uintreg_t *saved_regs;
```

Comment:
merge to previous patch?

Reply:
Yes, they both live in `arch/risc-v/include` so that is better.

Reply:
We need put this with the `riscv_dispatch_irq` changes to pass CI together, it shouldn't be merged to preivous one.

Reply:
if so, please move the typedef of uintreg_t to this patch

Reply:
ok